config/matomoWalletsEvents.ts

```diff
@@ -1,8 +1,16 @@
 import { MatomoEventType, trackEvent } from '@lidofinance/analytics-matomo';
 import { Metrics as WalletsMetrics } from 'reef-knot';
 
+// Should be change for Lido on X project!!!
```

Comment:
@solidovic 
That's not grammatically correct.
"Should be change**d**".
But I would write something like
"Please, change the string to be relevant to your project. For example: ..."

Reply:
done